README.md

```diff
@@ -122,14 +122,27 @@ You can deploy this project using: github, VS Code
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 👥 Author <a name="Vivek Raj"></a>
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author 1**
+
 
 👤 **Vivek Raj**
 
 - GitHub: [@githubhandle](https://github.com/VivekRaaZ?tab=repositories)
 - Twitter: [@twitterhandle](https://twitter.com/BossVivekRaj)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/vivek-raj-a78b92256/)
 
+  👤 **Author 2**
+
+👤 **Okeke Okechukwu**
+
+- GitHub: [@githubhandle](https://github.com/okayboy12)
+- Twitter: [@twitterhandle](https://twitter.com/okayboy)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/okayboy)
```

Comment:
Nice job updating your readme file😉. Currently, the link to your `mit` license is broken. kindly do well to fix it as this is giving your project a less professional look👍🏼.

<img width="1077" alt="image" src="https://github.com/VivekRaaZ/portfolio/assets/64643995/07483e60-a643-40be-87f0-ef43f36f6c10">